circuit/src/builder/circuit_builder.rs

```diff
@@ -67,6 +67,22 @@ where
         self.config.enable_mmcs(mmcs_config);
     }
 
+    /// Enables HashAbsorb operations.
+    pub fn enable_hash_absorb(&mut self, reset: bool) {
+        self.config.enable_hash_absorb(reset);
+    }
+
+    /// Enables HashSqueeze operations.
+    pub fn enable_hash_squeeze(&mut self) {
+        self.config.enable_hash_squeeze();
+    }
+
+    /// Enables hash operations.
+    pub fn enable_hash(&mut self, reset: bool) {
+        self.enable_hash_absorb(reset);
+        self.enable_hash_squeeze();
+    }
+
```

Comment:
Technically unrelated but I noticed these missing. I haven't kept track of the latest discussions around hashing handling so can remove these changes if needed.